### PR TITLE
[script.module.ssearlo] 0.0.22

### PR DESCRIPTION
**Pull request by JavaWiz1**
### Description
Clone of sseclient, locking on v0.0.22 due to bugs in v0.0.24.  I've named this ssearlo to isolate the code so the main package won't break the plugin again.

### NOTE:
3 items need to be merged at the same time

[script.module.ssearlo] 0.0.22
[script.module.arlo] 1.2.33
[plugin.video.arlowview] 1.1.0
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: script.module.ssearlo/addon.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    <import addon="xbmc.python" version="2.26.0"/>
+    <import addon="script.module.requests" version="2.9"/>
+    <import addon="script.module.six" version="1.11.0"/>
+  </requires>
+  <extension point="xbmc.python.module" library="lib" />
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_GB">Python client library for reading Server Sent Event streams based on sseclient.</summary>
+    <description lang="en_GB">Cloned from from https://github.com/btubbs/sseclient 0.0.22 due to bug in 0.0.24</description>
+    <platform>all</platform>
+    <license>MIT License</license>
```
Comment (enen92) [inline script.module.ssearlo/addon.xml]: MIT please

Comment (enen92) [reply script.module.ssearlo/addon.xml]: (spdx identifier)

Comment (JavaWiz1) [reply script.module.ssearlo/addon.xml]: Sorry for my ignorance, but I'm not sure what your comments mean.  As required, there is a LICENSE file with the MIT text in it (cloned from the original author), and I'm not sure what spdx identifier is...

Comment (enen92) [reply script.module.ssearlo/addon.xml]: I mean to replace the "MIT License" text with just "MIT". It's the spdx identifier for the license: https://spdx.org/licenses/

**Comment by enen92** [comment]
Please fix your pull request. You're including files from script.metadata.editor.

**Comment by JavaWiz1** [comment]
Little lost as to how that happened.  Would the proper way to correct be - 
git checkout origin/leia script.metadata.editor/  
(i.e. revert back to version in the leia branch)?

**Pull request merged**